### DISCUSSION TITLE
Seting up my own dgs-supported API Gateway / integration not clear

### DISCUSSION THREAD

**Discussion by tendomart** [Netflix/dgs-framework#407] [Q&A]
Hi team ,  i have  task of setting up dgs-Framework to interface between my already exsiting EMR (OpenMRS Rest API to be specific) . But looking around the getting started guide and how to examples , has not provided any examples.

All i have found is setting up the server coupled with either database models .

 I have seen something to do with **Using DgsRequestData**  and  **@RequestHeader and @RequestParam**  in the **DataFetcher's** section

 Can someone show me a full example of interating an existing rest endpoint or swagger scema using dgs-framework ?

Thanks alot , regards Tendo

**Comment by paulbakker** [comment]
There is no way to automatically integrate with a REST API, that's something you would have to write code for.

**Comment by tendomart** [comment]
Thanks Paul , i wish there is some one who has solved something  similar so
i look at the example.
